pkgs/by-name/fi/filebot/package.nix

```diff
@@ -36,6 +41,7 @@ stdenv.mkDerivation (finalAttrs: {
   nativeBuildInputs = [
     makeWrapper
     autoPatchelfHook
+    zenity
```

Comment:
Filebot has a configuration to enable zenity as the file browser and is the current recommended way according to the author rednoah@discord. I added it so it can be used but I am not yet sure how to enable it by default.